.github/workflows/review.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: GPTcode-reviewer
-        uses: PierreGode/GPTcode-reviewer@main
+        uses: PierreGode/GPTcode-reviewer@dev
```

Comment:
Using a `dev` branch instead of `main` could lead to instability. Ensure that the `dev` branch is stable and tested before using it in production workflows.

Reply:
looks good